.github/workflows/release.yml

```
@@ -60,11 +61,12 @@ jobs:
           sed -i 's/:v[0-9]*.[0-9]*.[0-9]*/:${{ github.event.release.tag_name }}/' docs/docs/documentation/getting-started/installation/sqlite.md
           sed -i 's/:v[0-9]*.[0-9]*.[0-9]*/:${{ github.event.release.tag_name }}/' docs/docs/documentation/getting-started/installation/postgres.md
 
-      - name: Commit updates
-        uses: test-room-7/action-update-file@v1
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
         with:
-          file-path: |
-            docs/docs/documentation/getting-started/installation/sqlite.md
-            docs/docs/documentation/getting-started/installation/postgres.md
-          commit-msg: "Change image tag, for release ${{ github.event.release.tag_name }}"
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "Change image tag, for release ${{ github.event.release.tag_name }}"
+          branch: "docs/newrelease-update-version-${{ github.event.release.tag_name }}"
+          delete-branch: true
+          base: mealie-next
+          title: "docs(auto): Change image tag, for release ${{ github.event.release.tag_name }}"
```

LGTM 
Really nitpicky, but i would use _Update_ instead of _Change_. Will leave it up to you if you want to update (or change 😁) that, or just merge as is. 

```suggestion
          commit-message: "Update image tag, for release ${{ github.event.release.tag_name }}"
          branch: "docs/newrelease-update-version-${{ github.event.release.tag_name }}"
          delete-branch: true
          base: mealie-next
          title: "docs(auto): Update image tag, for release ${{ github.event.release.tag_name }}"
```

---

I had no preference between them to be honest, so went with the suggestion

---

Looks like that gets rid of your approval, BTW